org.videolan.VLC.Plugin.makemkv.json

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "org.videolan.VLC.Plugin.makemkv",
+  "build-extension": true,
+  "runtime": "org.videolan.VLC",
+  "runtime-version": "stable",
+  "branch": "3-1.6",
```

Comment:
VLC isn't branched so your plugin shouldn't be either IMHO.

Reply:
@barthalion Sorry what do you mean by this ? The extension point in VLC is versionned so I need to target 3-18.08 (nevermind the 3-1.6 here, it's 3-18.08 in the current commit).

Reply:
Or you are talking about `runtime-version` ?
Without it on my computer it defaults to master and fails.